glide.lock

```diff
@@ -1,12 +1,12 @@
-hash: 8821a10afa7fa0a739161963d37154c7ed31841dc915a29978f2d27885acffbc
-updated: 2017-06-28T15:29:12.707326-04:00
+hash: e64cd277d54c83574b17c098f4e1e4631d185018b3171582d6de7c5e60fac307
+updated: 2017-07-22T22:13:22.426165653Z
 imports:
 - name: github.com/agl/ed25519
   version: 278e1ec8e8a6e017cd07577924d6766039146ced
   subpackages:
   - edwards25519
 - name: github.com/boltdb/bolt
-  version: 583e8937c61f1af6513608ccc75c97b6abdf4ff9
+  version: 2f1ce7a837dcb8da3ec595b1dac9d0632f0f99e8
```

Comment:
Please remove this and the grpc/protobuf update.  We're only interested in updating the other decred dependencies.